.babelrc-v9.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 10 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27036&commitId=c8127430f03d8160663561cd4f5349ee9ec320d5&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1037 screenshots unchanged, 0 screenshots with different dimensions and 8 screenshots with visible difference.


  <details><summary><b>unknown</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|FolderCover.Shared Cover.default.chromium.png|533| Changed|
|FolderCover.Small Media Cover.default.chromium.png|516| Changed|
|GroupedList.Root - RTL.click.chromium.png|87| Changed|
|GroupedList.Root - RTL.default.chromium.png|63| Changed|
|GroupedList.Root - RTL.hover.chromium.png|108| Changed|
|GroupedList.Root.click.chromium.png|87| Changed|
|GroupedList.Root.default.chromium.png|64| Changed|
|GroupedList.Root.hover.chromium.png|109| Changed|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>